.github/workflows/cross-bootstrap-tools.yml

```diff
@@ -15,11 +15,11 @@ jobs:
       matrix:
         include:
           # TODO: both Ubuntu and macOS have bmake packages, we should try them instead of bootstrapping our own copy.
-          - os: ubuntu-20.04
-            compiler: clang-10
-            cross-bindir: /usr/lib/llvm-10/bin
+          - os: ubuntu-16.04
+            compiler: clang-9
+            cross-bindir: /usr/lib/llvm-9/bin
             pkgs: bmake libarchive-dev
-          - os: macOS-latest
+          - os: macos-11.0
```

Comment:
11 not 11.0 (now following an iOS-like versioning scheme)

Reply:
Looks like the GitHub documentation is wrong then..

Reply:
Hm, https://github.com/actions/virtual-environments/issues/2486. So the name is right, but it's a terrible one done without any proper thought.